node-api/handlers/beacon/types/request.go

```diff
@@ -67,7 +67,7 @@ type GetValidatorBalancesRequest struct {
 
 type PostValidatorBalancesRequest struct {
 	types.StateIDRequest
-	IDs []string `validate:"dive,validator_id"`
+	IDs []string `json:"-" validate:"dive,validator_id"`
```

Comment:
what is `dive`?

Reply:
dive is a validation tag in go-playground/validator package- it is used to dive in array/map and validate each element inside it according to the validation rules.
In this case, go into slice `IDs` and apply `validator_id` validation rule at each string in the slice. Refer [dive](https://pkg.go.dev/github.com/go-playground/validator/v10#hdr-Dive)